Sources/TDS/Utilities/TDS+GeneralRules.swift

```diff
@@ -26,7 +29,7 @@ public typealias LongLong = Int64
 public typealias ULongLong = UInt64
 
 /// UCHAR
-public typealias UChar = Byte
+public typealias UChar = TDS.Byte
```

Comment:
For now lets leave this unscoped as it doesnt need specification in this context.